src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -33,12 +33,22 @@ public partial class ScriptEditorWindow : ModelessChildWindow
         public PythonNode nodeModel { get; set; }
         private bool nodeWasModified = false;
         private string originalScript;
-        public PythonEngineVersion CachedEngine { get; set; }
+
+        public string CachedEngine { get; set; }
 
         /// <summary>
         /// Available Python engines.
         /// </summary>
-        public ObservableCollection<PythonEngineVersion> AvailableEngines
+        [Obsolete()]
+        public ObservableCollection<PythonEngineVersion> AvailableEnginesd
```

Comment:
spelling (d?)

Reply:
Oh..missed that. I changed the name so I would not miss any references to it while refactoring.